code/modules/surgery/organs/stomach.dm

```diff
@@ -94,6 +104,16 @@
 	icon_state = "stomach-p"
 	desc = "A strange crystal that is responsible for metabolizing the unseen energy force that feeds plasmamen."
 
+/obj/item/organ/stomach/plasmaman/on_life()
```

Comment:
This should probably be in the milk definition rather than checking every tick for milk